frontend/projects/valtimo/components/src/lib/components/form-io/components/object-management-select/object-management-select.component.ts

```
@@ -0,0 +1,593 @@
+
+  private _formatSelectionValue(item: any): ObjectManagementSelectValue {
+    const source = item['_source'] as ObjectWrapper;
+
+    switch (this.valueFormat) {
+      case 'id':
+        return {id: source.uuid};
+      case 'full':
+        return {id: source.uuid, ...source};
+      case 'columns':
+      default:
+        const mapped: ObjectManagementSelectValue = {id: source.uuid};
```

Biome (2.5.1)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)